src/middleware/jwt/jwt.ts

```diff
@@ -46,11 +48,12 @@ export type JwtVariables<T = any> = {
  * ```
  */
 export const jwt = (options: {
-  secret: SignatureKey
+  secret: SignatureKey,
   cookie?:
     | string
     | { key: string; secret?: string | BufferSource; prefixOptions?: CookiePrefixOptions }
-  alg?: SignatureAlgorithm
+  alg?: SignatureAlgorithm,
+  header?: string,
```

Comment:
I think `headerName` is better than `header` with reference to the following Bearer Auth middleware.

https://github.com/honojs/hono/blob/882e75a3e00e98997177aaaef60bbf9b1b4eb3bf/src/middleware/bearer-auth/index.ts#L32